ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/test/helpers/ABMTestHelper.java

```diff
@@ -117,6 +118,51 @@ public static void deleteDataPolicy(
     }
 
     /**
+     * Deletes all buckets associted with a data policy
+     * to ensure that said policy can be deleted
+     * If any buckets using the policy are not properly deleted, an error is logged.
+     */
+    public static void deleteBucketsWithDataPolicy(
+            final String dataPolicyName,
+            final Ds3Client client) {
+        if (isEmpty(dataPolicyName)) {
+            //This might not be an error if this function is called as part of cleanup code
+            LOG.debug("Data policy name is null or empty");
+            return;
+        }
+        //Get all buckets using the data policy
+        try {
+            final GetBucketsSpectraS3Response bucketsResponse = client
+                    .getBucketsSpectraS3(new GetBucketsSpectraS3Request().withDataPolicyId(dataPolicyName));
+
+            final BucketList bucketList = bucketsResponse.getBucketListResult();
+
+            for (final Bucket bucket : bucketList.getBuckets()) {
+                // delete each
+                final DeleteBucketSpectraS3Response deleteBucketSpectraS3Response = client
+                        .deleteBucketSpectraS3(new DeleteBucketSpectraS3Request(bucket.getName()));
+                assertThat(deleteBucketSpectraS3Response.getStatusCode(), is(204));
+            }
+
+        } catch (final IOException|AssertionError e) {
+            LOG.error("Bucket assigned to data policy was not deleted as expected: " + dataPolicyName, e);
+        }
+
+        // Verify that no buckets are attached to the data policy
+        try {
+            final GetBucketsSpectraS3Response bucketsResponse = client
+                    .getBucketsSpectraS3(new GetBucketsSpectraS3Request().withDataPolicyId(dataPolicyName));
+
+            final BucketList bucketList = bucketsResponse.getBucketListResult();
+            if (!Guard.isNullOrEmpty(bucketList.getBuckets())) {
+                LOG.error("Buckets using data policy still exist despite deletion attempt");
+            }
+        } catch (final IOException e) {
+            LOG.error("Failed getting buckets to verify none use data policy " + dataPolicyName, e);
```

Comment:
Use `{}` to be consistent with the new way we are doing logging.  So this needs to be updated to:

```java
LOG.error("Failed getting buckets to verify none use data policy {}", dataPolicyName, e);
```